packages/kokoas-client/src/pages/projSearch/sections/result/details/projectDetails/ProjectDetails.tsx

```diff
@@ -14,43 +14,60 @@ import { OfficerDetails } from './OfficerDetails';
 const Address = ({
   postal,
   address,
-}:{
+}: {
   postal: string,
   address: string,
 }) => {
   return (
     <>
       {postal && (
-      <div>
-        {postal}
-      </div>)}
+        <div>
+          {postal}
+        </div>)}
       {address || '-'}
     </>
   );
 };
 
+const Memo = (({
+  value,
+}: {
+  value: string,
+}) => {
+  const splitVal = value.split('\n');
+  return (
+    <>
+      {splitVal.map((line, index) => (
+        <div key={`remarks-${line}`}>
+          {line}
+          {index < (splitVal.length - 1) && <br />}
```

Comment:
変更不要です！

以下もいけるかもしれません。そうでなければ、教えてください。

```suggestion
          {!!index && <br />}
```

Reply:
ご指摘ありがとうございます。
上記、条件が不要なことがわかりましたので、条件なしで改行を入れるようにします。
また、口頭でもお伝えしましたが、1行ずつdivで囲んでしまっていたため、修正しました。